next/schema-generator/src/generator/helpers.ts

```diff
@@ -180,3 +182,33 @@ export const createCamelCaseOptionsV2 = <Option extends { title: string }>(
   assertUniqueOptions(result)
   return result
 }
+
+/**
+ * Create phone number input field consitent with all forms.
+ */
+export const inputPhoneNumber = (property: string, required: boolean, helptext?: string) =>
+  input(
+    property,
+    { title: 'Telefónne číslo', required, type: 'tel' },
+    { size: 'medium', placeholder: '+421', helptext },
+  )
+
+/**
+ * Create address input fields.
+ */
+export const inputAddress = (property: string, title: string, required: boolean) =>
```

Comment:
Also, I would name this `addressField` or `sharedAddressField`, as it is more consistent with what we already have.